yoyodyne/models/base.py

```diff
@@ -295,12 +323,21 @@ def predict_step(
             batch_idx (int).
 
         Returns:
-            torch.Tensor: indices of the argmax at each timestep.
+            Tuple(torch.Tensor, torch.Tensor): position 0 are the indices of 
+            the argmax at each timestep. Position 1 are the scores for each 
+            history in beam search. It will be None when using greedy.
+
         """
         predictions = self(batch)
-        # -> B x seq_len x 1.
-        greedy_predictions = self._get_predicted(predictions)
-        return greedy_predictions
+
```

Comment:
Pet peeve of mine but I hate blank lines within functions; they make it hard for me to tell where a function starts or ends. Add a comment or break it into smaller functions if you feel the need. A bunch were added here, can you remove them?